geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/ColorMap.scala

```diff
@@ -0,0 +1,51 @@
+package geopyspark.geotrellis
+
+import geotrellis.raster.render
+
+import scala.collection.JavaConverters._
+import scala.collection.JavaConversions._
+
+object ColorMap {
+
+  def stringToBoundaryType(s: String) = s.toLowerCase match {
```

Comment:
We already have a method that does this https://github.com/locationtech-labs/geopyspark/blob/master/geopyspark-backend/geotrellis/src/main/scala/geopyspark/geotrellis/GeoTrellisUtils.scala#L83